spec/controllers/admin/audit_logs_controller_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe Admin::AuditLogsController, type: :controller do
```

Comment:
No need to add a blank test. Instead, I'd love to see a feature spec that tests the search.

Reply:
Added feature tests and deleted this file.